demos/travel-and-hospitality/alliance-ticket-sharing/uni_configuration/schema.json

```diff
@@ -0,0 +1,558 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "$id": "https://vendia.net/schemas/alliance-ticket-sharing/v1.0",
+  "title": "Alliance Partner Ticket Share",
+  "description": "A ticket data model with ACLs that allows alliance partners to share ticket data in real-time and with control",
+  "x-vendia-acls": {
+    "TicketAcl": {
+      "type": "Ticket"
+    }
+  },
+  "type": "object",
+  "properties": {
+    "Ticket": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "passengerNameRecordCode": {
+            "type": "string",
+            "description": "A unique passenger record, represented by a 6-digit alphanumeric value",
+            "pattern": "[A-Z0-9]{6}"
+          },
+          "airlineCode": {
+            "type": "string",
+            "description": "The 2-letter code of the issuing airline",
+            "pattern": "[A-Z]{2}"
+          },
+          "ticketNumber": {
+            "type": "string",
+            "description": "A unique ticket number, represented as a 3-digit airline code followed by a 4-digit form number followed by a 6-digit serial number",
+            "pattern": "[0-9]{13}"
+          },
+          "dateOfIssue": {
+            "type": "string",
+            "description": "The date and time this ticket was issued, in the format YYYY-MM-DD",
+            "format": "date"
+          },
+          "bookingAgentId": {
+            "type": "string",
+            "description": "A code identifying the booking agent, as determined by the issuing airline"
+          },
+          "remarks": {
+            "type": "string",
+            "description": "Free text to provide further information related to the ticket"
+          },
+          "passenger": {
+            "type": "object",
+            "description": "Information about the passenger",
+            "properties": {
+              "firstName": {
+                "type": "string",
+                "description": "Passenger's first name"
+              },
+              "lastName": {
+                "type": "string",
+                "description": "Passenger's last name"
+              },
+              "dateOfBirth": {
+                "type": "string",
+                "description": "The date of birth of the passenger, in YYYY-MM-DD format",
+                "format": "date"
+              },
+              "frequentFlyerInformation": {
+                "type": "object",
+                "description": "Passenger's frequent flyer information",
+                "properties": {
+                  "airlineCode": {
+                    "type": "string",
+                    "description": "The 2-letter code of the airline",
+                    "pattern": "[A-Z]{2}"
```

Comment:
Same question about the character class only allowing upper case letters.

Reply:
Intentional